src/orchard/java.clj

```diff
@@ -345,26 +345,36 @@
           (first ms)
           {:candidates (zipmap (map :class ms) ms)})))))
 
+(def backported-javadoc-bases
```

Comment:
Probably we can name this `javadoc-base-urls`. I think `backported` is a bit confusing in this context.